docs/source/traitsui_user_manual/testing/substitutions.rst

```diff
@@ -20,7 +20,6 @@
 .. |TargetRegistry.register_location| replace:: :class:`~traitsui.testing.tester.target_registry.TargetRegistry.register_location`
 
 .. |UITester| replace:: :class:`~traitsui.testing.tester.ui_tester.UITester`
-.. |UITester.auto_process_events| replace:: :attr:`~traitsui.testing.tester.ui_tester.UITester.auto_process_events`
```

Comment:
Please leave this alone, we can add `auto_process_events` in the Attributes section in `UITester`. https://github.com/enthought/traitsui/blob/7a960c68d65bfd2b76a857ef6c4c0deffa256550/traitsui/testing/tester/ui_tester.py#L48-L49

That is why the link does not work. And on the other hand, this is a bit subjective, I think the bolded font for an attribute that does not have a cross reference still looks nicer than the one with double backticks.

Reply:
reverted